docs/tutorial/using-checkbox/test-report.rst

```diff
@@ -0,0 +1,87 @@
+.. _test-report:
+
+Review Test Report 
+======================
+
+Checkbox optionally generates test reports in different formats that can be used to easily share the results of a test session. Generally, a :ref:`text summary<test-summary>` and :ref:`submission files<submission-files>` would be created. In this section, you will learn where to find the report files and what they contain.
+
+Note that you can tailor desired reports in your launcher file, or define your own exporter to customize the reports. See :doc:`launcher<../../reference/launcher>` and :doc:`exporter<../../reference/units/exporter>` for more details.
+
+.. _test-summary:
+Text summary
+------------
+
+A text summary shows in the console once all jobs complete, providing an overview of test result. You can find the result of each job displaying in a form of ``outcome: summary``.
```

Comment:
I would rephrase this as: A text summary is displayed in the console once all jobs are completed. Each result is in the form: ``outcome: summary``.

Reply:
fixed in [fdd5e20](https://github.com/canonical/checkbox/pull/637/commits/fdd5e208e18dfabfacf2c8bf416d1b017b73251b)